runtime/codert_vm/jswalk.c

```diff
@@ -1541,7 +1541,20 @@ J9JITExceptionTable * jitGetExceptionTableFromPC(J9VMThread * vmThread, UDATA ji
 	return jit_artifact_search(vmThread->javaVM->jitConfig->translationArtifacts, maskedPC);
 }
 
+BOOLEAN jitIsFSDMethodBody(J9VMThread * vmThread, J9Method * method)
```

Comment:
This does not belong in the stack walker - it belongs in MethodMetaData and should take the metadata pointer as a parameter rather than the method. I already need to do the compiled check in the decompiler.

Reply:
Alternately, I could just test the bit myself (perhaps a query is overkill for something so simple, it just guards us against any future encoding changes).

Reply:
Yeah I suppose if the caller of this query is going to provide the `J9JITExceptionTable`, then the caller could test the flag directly. I can make the appropriate changes.

Reply:
Updated in [force push](https://github.com/eclipse-openj9/openj9/compare/7283e98ed2e214b33df5fd9c5be19ec623362daf..eef4d55a8e6291baf82a13cac5faaa12787deb6b).